### ISSUE TITLE
still can't reproduce the accuracy

### ISSUE THREAD

**Issue by qzhai** [RenqiChen/Genomic-Selection#9]
Hi, thank you for the updated comment! However, I’m still unable to reproduce the accuracy reported in the paper. Would you mind kindly sharing the pretrained model parameters? I’d greatly appreciate it. Thank you!

**Comment by RenqiChen** [comment]
Hello, it is a end-to-end training framework, where there are no pretrained model parameters. Could you tell us the accuracy you have achieved?

**Comment by qzhai** [comment]
Hi, what I mean by the pre-trained models is that you released them to report the final result. I found that all the classification accuracies (PTH, PEX_REPRO, LSEN, LPCO_REV_POST, CUST_REPRO, APCO_REV_REPRO) are lower than those reported in the paper, e.g., I got a final accuracy of 0.508 for PTH, which is even lower than DLGWAS. So I would greatly appreciate it if the final parameters could be released for reference. Thank you very much!

**Comment by RenqiChen** [comment]
Do you follow the settings in the paper? Because your results are much lower than ours.

**Comment by qzhai** [comment]
I believe there should not be a significant difference. I have carefully compared the paper with the code, as well as the details I confirmed previously.

**Comment by RenqiChen** [comment]
We understand it. To resolve your problems, we will release our pre-trained model parameters soon.

**Comment by qzhai** [comment]
So thank you so much.
